packages/typespec-ts/test/unit/modelsGenerator.spec.ts

```diff
@@ -283,6 +283,29 @@ describe("Input/output model type", () => {
           true
         );
       });
+
+      it("should handle enum member", async () => {
+        const tspTypeDefinition = `
+        #suppress "@azure-tools/typespec-azure-core/use-extensible-enum" "for test"
+        @fixed
```

Comment:
I think the fixed or extensible would not impact the enum member generation, right?

Reply:
yeah, I just copy other test case.